content/blog/2020/08/2020-08-27-winsw-yaml-support.adoc

```diff
@@ -123,21 +143,6 @@ If there are both XML and YAML configuraiton files, Windows Service Wrapper will
 
 video::9qyo1f2rKQw[youtube, start=2736, end=4000, width=640, height=360]
 
-== Future Works
```

Comment:
It would be nice to keep it and to list the originally planned items which have not been delivered